packages/presentation/lib/di/injector.dart

```diff
@@ -0,0 +1,6 @@
+// import 'package:get_it/get_it.dart';
+// import 'package:injectable/injectable.dart';
+// import 'package:presentation/di/injector.config.dart';
+
+// @InjectableInit()
+// void configurePresentationDependencies(GetIt getIt) => $initGetIt(getIt);
```

Comment:
you don't need to bring it to master branch as it doesn't related to clear impl of getit